Assets/Scripts/Character/Movement/Jump.cs

```diff
@@ -0,0 +1,83 @@
+using UnityEngine;
+
+[RequireComponent(typeof(Rigidbody2D))]
+public class Jump : PlayableBehaviour<Jump.Context>
+{
+    public class Context
+    {
+        public float jumpSpeed;
+
+        public Context(float jumpSpeed)
+        {
+            this.jumpSpeed = jumpSpeed;
+        }
+    }
+
+    private Rigidbody2D rb;
+
+    public float maxJumpTimeSeconds;
+    public override bool Playing => Jumping;
+
+    private static readonly int JumpingHash = Animator.StringToHash("jumping");
+
+    private float startTime;
+    private bool jumping;
+
+    //helps determine when the first jump call was made
+    private bool startedJump;
```

Comment:
Maybe change the design here a little be:
* Play should be called once from outside, and turn on some flag
* Update here should check the flag and add speed
* Add a public method `StopAddingSpeed` (bad name) that turns the flag off
This way this class is not coupled to our input controller